src/parser/parse_tree_cl.c

```diff
@@ -6592,7 +6592,8 @@ pt_print_attr_def (PARSER_CONTEXT * parser, PT_NODE * p)
       if (p->data_type)
 	{
 	  /* only show non-default parameter */
-	  if (p->data_type->info.data_type.precision != 15 || p->data_type->info.data_type.dec_precision != 0)
+	  if (p->data_type->info.data_type.precision != DB_DEFAULT_NUMERIC_PRECISION
+	      || p->data_type->info.data_type.dec_precision != DB_DEFAULT_NUMERIC_SCALE)
```

Comment:
The condition of this if statement is strange.
Isn't that always true?

Reply:
This is true only for NUMERIC (15, 0) type.

Reply:
Do you think it should be '&&' instead of '||'?

Reply:
All right. I was looking at the code wrong
You were comparing different variables with precision and dec_precision.
I mistakenly thought it was the same variable.

Reply:
It is confusing because it is not called scale here, but dec_precision.